src/state/state.js

```diff
@@ -188,14 +188,14 @@
          * @ignore
          */
         function _renderFrame(time) {
-            // update all game objects
-            me.game.update(time);
-            // render all game objects
-            me.game.draw();
             // schedule the next frame update
             if (_animFrameId !== -1) {
                 _animFrameId = window.requestAnimationFrame(_renderFrame);
             }
+            // update all game objects
+            me.game.update(time);
+            // render all game objects
+            me.game.draw();
```

Comment:
This code change cannot change the result, because JavaScript is single-threaded and *uninterruptible*. The only way to get smoother animations to to make sure the `draw()` *and* `update()` methods complete within the ~16.667ms timeframe (for 60FPS). The order of operations cannot speed it up. ;)